Automattic/wp-calypso
# Adding new launchpad flow for VideoPress!

#### Proposed Changes

* Adds a new launchpad checklist for VideoPress/Videomaker onboarding:

<img width="1376" alt="Screen Shot 2022-09-16 at 11 33 39 AM" src="https://user-images.githubusercontent.com/789137/190699152-0143408c-f389-477d-8bbb-fd46e47ef793.png">

* Related: D87583-code.

#### Testing Instructions

* Apply D87583 to your sandbox.
* Start with http://calypso.localhost:3000/setup/intro?flow=videopress, and complete a signup for a site.
* You should end up at the launchpad. Note that the active step is `Upload your first video`.
* Upload a video to the site.
* Return to the launchpad, and the step should be marked as completed.
* Click `Launch Video Site`, it should work properly.

#### Pre-merge Checklist
